talk about dandi staging (#33)
dandi/handbook: docs/10_using_dandi.md

        1. Convert your data to NWB 2.1+ in a local folder. Let's call this `<source_folder>`
        This step can be complex depending on your data. Feel free to [reach out to
        us for help](https://github.com/dandi/helpdesk/discussions).

>         2. Validate the NWB files by running: `dandi validate <source_folder>`. 

Author: ```suggestion
        1. Validate the NWB files by running: `dandi validate <source_folder>`. 
```

        **If you are having trouble with validation, make sure the conversions were run with the most recent version of `PyNWB` and `MatNWB`.**

>         3. Preparing a dataset folder for upload:

Author: ```suggestion
        1. Preparing a dataset folder for upload:
```

        us for help](https://github.com/dandi/helpdesk/discussions).
        1. Validate the NWB files by running: `dandi validate <source_folder>`. 
        **If you are having trouble with validation, make sure the conversions were run with the most recent version of `PyNWB` and `MatNWB`.**
        3. Preparing a dataset folder for upload:

                dandi download https://dandiarchive.org/dandiset/<dataset_id>/draft
                cd <dataset_id>
                dandi organize <source_folder> -f dry
                dandi organize <source_folder>
                dandi upload  # for staging: dandi upload -i dandi-staging

>         5. Add metadata by visiting your dandiset landing page: 
>        `https://dandiarchive.org/dandiset/<dataset_id>/draft` and clicking on the `METADATA` link.
>         6. Use the dandiset URL in your preprint directly, or download it using the dandi CLI:

Author: ```suggestion
        1. Add metadata by visiting your dandiset landing page: 
       `https://dandiarchive.org/dandiset/<dataset_id>/draft` and clicking on the `METADATA` link.
        1. Use the dandiset URL in your preprint directly, or download it using the dandi CLI:
```
